### PR TITLE
LPS-190202 Handle the inert attribute on the main wrapper when the simulation panel is opened or closed

### PR DESCRIPTION
**Pull request by veroglez**
https://liferay.atlassian.net/browse/LPS-190223

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

File: modules/apps/product-navigation/product-navigation-simulation-device/src/main/resources/META-INF/resources/js/components/Preview.tsx [hunk 1]
```diff
@@ -27,8 +27,22 @@ export default function Preview({activeSize, previewRef}: IPreviewProps) {
...
+			setVisible(false);
+
+			if (wrapper) {
+				wrapper.removeAttribute('inert');
+			}
+		};
+		const showIframe = () => {
+			setVisible(true);
+
+			if (wrapper) {
```
Comment (victorg1991) [inline modules/apps/product-navigation/product-navigation-simulation-device/src/main/resources/META-INF/resources/js/components/Preview.tsx:42]: I was very confued about how this had to do with the iframe, but it doesn't

Can we rename this method ? something like `onOpenSimulationPanel`?

**Comment by victorg1991** [comment]
Nooice :clap:

**Comment by victorg1991** [comment]
ci:forward

**Comment by veroglez** [comment]
ci:forward por favoh

**Comment by victorg1991** [comment]
image

**Pull request closed without merging**